golang/src/user-sessions/main.go

```diff
@@ -0,0 +1,393 @@
+package main
+
+import (
+	"configurate"
+	"database/sql"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"logcabin"
+	"net/http"
+	"os"
+	"queries"
+	"strings"
+
+	"github.com/gorilla/mux"
+	_ "github.com/lib/pq"
+)
+
+var (
+	version = flag.Bool("version", false, "Print the version information")
+	cfgPath = flag.String("config", "/etc/iplant/de/jobservices.yml", "The path to the config file")
```

Comment:
I'm hoping to get as much of the configs as I can into a single file. Right now that file is jobservices.yml, but it will probably get renamed to something more generic in the future. For now, I'd prefer to leave it.